apps/prairielearn/python/prairielearn.py

```diff
@@ -726,19 +727,17 @@ def get_color_attrib(element, name, *args):
         if val is None:
             return val
 
-        named_color = colors.get_css_color(val)
-        if named_color is not None:
-            return named_color
+        if Color.match(val) is not None:
+            return Color(val).to_string(hex=True)
```

Comment:
In the other Python files, I just left it as `Color(name)` (there was no reason to convert to a hex string), but here I left the contract the same (in case any callers depend on a hex string).